jruby/neo4j/driver.rb

```diff
@@ -38,5 +13,21 @@ module Net
     module Types
       include_package 'org.neo4j.driver.v1.types'
     end
+
+    def self.after_zeitwerk_load_complete
```

Comment:
we needed to execute following code snippet only after zeitwerk load is complete, or else the classes aren't available and unknown constant error is thrown. Hence added this code snippet in `after_zeitwerk_load_complete` to ensure we only call it after the zeitwerk load is complete.